rise/ViewControllers/LoginViewController.m

```diff
@@ -23,6 +24,12 @@ @implementation LoginViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    NotificationManager *notificationManager = [NotificationManager new];
+    [notificationManager requestAuthorization:^(BOOL granted) {
+        if (granted) {
+            NSLog(@"Notifications authorization granted.");
+        }
+    }];
```

Comment:
The good practice is checking if the authorization had been granted already before requesting.